gcp/gcp.go

```diff
@@ -487,6 +487,93 @@ const (
 	Owner   Role = "OWNER"
 )
 
+// Search calls google.com/cloudprint/search to query printers owned by the user/robot.
+func (gcp *GoogleCloudPrint) Search(q, atype, connection_status string) (map[string]lib.Printer, error) {
```

Comment:
More descriptive names than q, atype. Perhaps query and printer_type.

printer_type should be more structured, maybe a type with const values (enum in other languages) like this:
https://github.com/google/cloud-print-connector/blob/master/cdd/cdd.go#L157